xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-lightbox/xwiki-platform-image-lightbox-webjar/src/main/webjar/lightbox.js

```diff
@@ -173,17 +173,21 @@ define('xwiki-lightbox', [
    * Make sure that the toolbar will remain open also while hovering it, not just the image.
    */
   var keepToolbarOpenOnHover = function() {
+    var hideTimeout;
```

Comment:
Why this var declaration on a separate line?

Reply:
Because I have 2 places where I use the variable